src/tools/rest-backend/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+find_package (CppCMS)
+find_package (Boost 1.50 COMPONENTS system filesystem thread REQUIRED)
+
+if (NOT CppCMS_FOUND)
+    message(SEND_ERROR "CppCMS package not found")
```

Comment:
Please do not use `SEND_ERROR`, but instead use `remove_tool` as done some lines below.

To make the removal of tools/bindings/plugins more visible is a general issue and should be solved once for all tools/bindings/plugins. Please create a new issue if you have some ideas about what a user would expect.

Btw. all warnings should be solved in the PR before further review, too.